### PR TITLE
Translate CHANGELOG to Russian

### PR DESCRIPTION
**Pull request by emartynov**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR is part of issue #322.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by commjoen** [comment]
Thank you @emartynov !!! Let me check for a second russian speaker to do the translation review 👍

File: Document-ru/CHANGELOG.md [hunk 1]
```diff
@@ -1,57 +1,57 @@
-# Changelog
+# Список изменений
 
-This document is automatically generated at {{gitbook.time}}
+Этот документ автоамитчески сгенерирован {{gitbook.time}}
 
-## V1.1.4 4 July 2019 - Summit edition
+## V1.1.4 - 4 Июля 2019 - Саммит версия
```
Comment (Gaket) [inline Document-ru/CHANGELOG.md]: "Summit" probably should not be translated. @commjoen, what's the semantics of "summit" here?

File: Document-ru/CHANGELOG.md [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
-- Translated the changelog to Chinese (ZHTW) and Japanese.
-- Automated verification of the markdown syntax and reachability of the URLs.
-- Added identification codes to the requirements, which will be included in the future version of the MSTG in order to find the recommendations and testcases easily.
-- Reduced the repo size and added Generated to the .gitignore.
-- Added a Code of Conduct & Contributing guidelines.
-- Added a Pull-Request template.
-- Updated the sync with the repo in use for hosting the Gitbook website.
-- Updated the scripts to generate XML/JSON/CSV for all the translations.
-- Translated the Foreword to Chinese (ZHTW).
+- Исправлены все markdown ошибки.
```
Comment (Gaket) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Исправлены все ошибки markdown.
```

File: Document-ru/CHANGELOG.md [hunk 3]
```diff
@@ -1,57 +1,57 @@
...
+- Обновления во французском и испанском переводах.
+- Список изменений переведен на китайский и японский языки.
+- Автоматизированна проверка markdown синтаксиса и открытие ссылок в документах.
```
Comment (Gaket) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Автоматизирована проверка markdown синтаксиса и доступность ссылок в документах.
```

File: Document-ru/CHANGELOG.md [hunk 4]
```diff
@@ -1,57 +1,57 @@
...
+- Для облегчения поиска рекомендаций и тест-кейсов добавлены идентификационные номера требований, которые будут включены в следующих версиях MSTG.
+- Уменьшен размер репозитория и папка Generated добавлена в .gitignore.
+- Добавлен Свод Правил & Рекомендации по внесению изменений.
```
Comment (Gaket) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Добавлен Кодекс Поведения и Рекомендации по внесению изменений.
```

File: Document-ru/CHANGELOG.md [hunk 5]
```diff
@@ -1,57 +1,57 @@
...
-## V1.1.3 9 January 2019 - Small Fixes
+## V1.1.3 - 9  Января 2019 - Мелкие Изменения
 
-The following changes are part of release 1.1.3:
+Изменения в релизе 1.1.3:
 
-- Fix translation issue of requirement 7.1 in the Spanish version.
-- New setup of translators in acknowledgements.
-- Small fixes for the Japanese version.
+- Исправелена ошибка перевода требования 7.1 на испанский язык.
```
Comment (Gaket) [inline Document-ru/CHANGELOG.md]:
```suggestion
- Исправлена ошибка перевода требования 7.1 на испанский язык.
```

**Comment by Gaket** [comment]
Hi! Went through the translation, found a few small issues. It's always better to double check things :)

File: Document-ru/CHANGELOG.md (hunk 3 above)
Comment (emartynov) [reply Document-ru/CHANGELOG.md]: Good catch!

**Comment by emartynov** [comment]
Thanks! All are valuable. Applying them on the fly!

**Comment by emartynov** [comment]
Sorry for the many commits - just lazy to open an editor and apply them in a more natural and organised way.

File: Document-ru/CHANGELOG.md (hunk 1 above)
Comment (commjoen) [reply Document-ru/CHANGELOG.md]: thank you for your review @Gaket! So summit, i mean to refer to the open security summit: https://open-security-summit.org/

Comment (emartynov) [reply Document-ru/CHANGELOG.md]: I found in the translator that summit is already adopted Russian word. And it means a meeting or people gathering to discuss something.

Comment (emartynov) [reply Document-ru/CHANGELOG.md]: @Gaket Here is proof https://ru.wikipedia.org/wiki/%D0%A1%D0%B0%D0%BC%D0%BC%D0%B8%D1%82

**Comment by commjoen** [comment]
@emartynov  & @Gaket , can i merge this now given the conversations and commits?

**Comment by emartynov** [comment]
I would say - yes, please

**Comment by commjoen** [comment]
this one has been merged already :). are you on slack @kysokzla

**Comment by kysokzla** [comment]
@commjoen sorry man, then it should be fixed in next commit :) ill try to login

**Comment by commjoen** [comment]
no worries ;)

**Comment by emartynov** [comment]
I just copied the titles of the requirements from the already translated document:

https://github.com/OWASP/owasp-masvs/blob/master/Document-ru/0x07-V2-Data_Storage_and_Privacy_requirements.md

Unfortunately, I didn't check if it was already good enough.

**Comment by commjoen** [comment]
no worries @emartynov , can you pick it up when you work on https://github.com/OWASP/owasp-masvs/issues/322?

**Pull request merged**